src/flash/param.ts

```diff
@@ -287,6 +287,13 @@ export interface GetQuoteParams {
    * The EOA owner smart wallet (necessary for working with Smart Wallet)
    */
   fromActorReceiverWalletOwner?: string;
+
+  /**
+   * Max slippage for the swap in percentage
+   *
+   * @default 0
+   */
+  maxSlippagePct?: number;
```

Comment:
I beleive we already have some other params that are also percent yet don't have any suffixes

```suggestion
  maxSlippage?: number;
```

so you can also consider it to be `Amount` for consistency and do the `toDecimalString` for the API call

```suggestion
  maxSlippage?: Amount;
```